pkg/intermediate/aggregate.go

```diff
@@ -128,6 +128,13 @@ func (a *AggregationProcess) Stop() {
 	a.stopChan <- true
 }
 
+// GetNumOfConnections returns total number of connections stored in map
+func (a *AggregationProcess) GetNumOfConnections() uint64 {
```

Comment:
Prefer `GetNumberOfFlows` as we use connections in exporter and collector.

Reply:
Sure. I didn't see any. Thanks!